source/issue-a-credential/credential/index.html.md.erb

```diff
@@ -253,161 +260,82 @@ The example below shows the structure of a verifiable credential using a JSON We
+| `type`              | Required             | A set of values indicating the type of verifiable credentials issued by the issuer. The first value in the set must be `VerifiableCredential`.                                                                                                                                                                                                                                        |
+| `description`       | Optional             | Issuer-specified credential description.                                                                                                                                                                                                                                                                                                                                              |
+| `exp`               | Optional             | Expiration time. The time after which the JWT must not be accepted for processing. Must be expressed in epoch time as per the **sausages**                                                                                                                                                                                                                                            |
+| `iat`               | Optional             | Issued at. The time the JWT was issued. Must be expressed in epoch time as per the **sausages**.                                                                                                                                                                                                                                                                                      |
+| `name`              | Optional             | Issuer-specified credential name.                                                                                                                                                                                                                                                                                                                                                     |
+| `validFrom`         | Optional             | Represents the date and time the credential becomes valid, and the same as the value of the `nbf/iat` claims. Must be expressed in ISO 8601 format as per the [VC data model v2.0](https://www.w3.org/TR/vc-data-model-2.0/).                                                                                                                                                         |
+| `validUntil`        | Optional             | Represents the date and time the credential stops being valid. This value specifies the date until which the information within the `credentialSubject` property remains valid. In the example above, the values of `expirationDate` and `validUntil` are the same. Must be expressed in ISO 8601 format as per the [VC data model v2.0](https://www.w3.org/TR/vc-data-model-2.0/).   |
```

Comment:
Is there a reason we're saying 'Represents' here, as opposed to just 'The date and time the credential...'?